MCDataProducts/inc/GenId.hh

```diff
@@ -43,7 +43,7 @@ namespace mu2e {
       CaloCalib, InFlightParticleSampler, muplusDecayGun, StoppedMuonXRayGammaRayGun, //37
       cosmicCRY,  pbarFlat, fromAscii, ExternalRMC, InternalRMC, CeLeadingLog, cosmicCORSIKA, //44
       MuCapProtonGenTool, MuCapDeuteronGenTool, DIOGenTool, MuCapNeutronGenTool, // 48
-      MuCapPhotonGenTool, MuCapGammaRayGenTool, CeLeadingLogGenTool,// 50
+      MuCapPhotonGenTool, MuCapGammaRayGenTool, CeLeadingLogGenTool, MuplusMichelGenTool,// 50
```

Comment:
Please update the comments so that they remain accurate.

Reply:
I re-write the comment. Now the GenID of 'MuPlusMichelGenTool' is 52 and 'LastEnum' is 53. I committed my changes locally. Should I try pull again?